tests/test_transactions_api.py

```diff
@@ -626,3 +630,42 @@ def test_span_id(self) -> None:
         assert response.status_code == 200, response.data
 
         assert data["data"][0]["span_id"] == "8841662216cc598b"
+
+    def test_limitby_multicolumn(self) -> None:
+        query_str = """MATCH (transactions)
+                    SELECT project_id,
+                           environment,
+                           platform,
+                           event_id
+                    WHERE project_id = 1
+                    AND finish_ts >= toDateTime('{start_time}')
+                    AND finish_ts < toDateTime('{end_time}')
+                    LIMIT {limit_by_count} BY environment, platform
+                    """.format(
+            start_time=(self.base_time - self.skew).isoformat(),
+            end_time=(self.base_time + self.skew).isoformat(),
+            limit_by_count=LIMIT_BY_COUNT,
+        )
+        response = self.app.post(SNQL_ROUTE, data=json.dumps({"query": query_str}))
```

Comment:
```suggestion
        response = self.app.post(SNQL_ROUTE, data=json.dumps({"query": query_str, "dataset": "transactions"}))
```

Obviously this isn't strictly necessary but is best practice.